src/tree/SubscriptionTreeItem.ts

```diff
@@ -51,13 +57,20 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         const promptSteps: AzureWizardPromptStep<IStaticWebAppWizardContext>[] = [];
         const executeSteps: AzureWizardExecuteStep<IStaticWebAppWizardContext>[] = [];
 
-        promptSteps.push(new CreateScenarioListStep());
+        // because a workspace is a requirement, we can be confident that there is a fsPath
+        await WorkspaceListStep.setWorkspaceContexts(wizardContext, nonNullProp(wizardContext, 'fsPath'));
+
+        // if the local project doesn't have a GitHub remote, we will create it for them
+        if (!wizardContext.originExists) {
+            promptSteps.push(new GitHubOrgListStep(), new RepoNameStep(), new RepoPrivacyStep(), new RemoteShortnameStep());
+            executeSteps.push(new RepoCreateStep(), new GitignoreCreateStep());
+        }
 
         promptSteps.push(new StaticWebAppNameStep());
         if (context.advancedCreation) {
             promptSteps.push(new ResourceGroupListStep());
         } else {
-            const remoteRepo: ReposGetResponseData | undefined = await tryGetRemote(context);
+            const remoteRepo: ReposGetResponseData | undefined = await tryGetProjectForCreation(context);
```

Comment:
If I have two git projects open in my workspace when creating and I select the second one, this error is thrown:

<img width="385" alt="Screen Shot 2021-04-08 at 1 51 48 PM" src="https://user-images.githubusercontent.com/22795803/114094948-c0e6a700-9871-11eb-9cf6-47eee4eaa04d.png">

Reply:
Should be resolved by passing in the fspath